src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -38,6 +38,7 @@ public class SonarQubeIssueProviderTests
         private Mock<ISonarQubeService> mockSqService;
         private Mock<ITimerFactory> mockTimerFactory;
         private Mock<ITimer> mockTimer;
+        private TestLogger testLogger;
```

Comment:
I used the test logger rather than a mock because it's easier to check a series of expected messages.

Reply:
I don't mind using test classes :)